docs/adr/0007-azure-pipeline-security-model.md

```diff
@@ -0,0 +1,99 @@
+# Azure pipeline security model
+
+## Status
+
+Proposed
+
+
+## Context
+
+This ADR defines an approach for balancing the principle of least-privilege with the benefits from having a complete, repeatable and self-healing pipeline.  Specifically, it relates to Azure DevOps Pipelines using Microsoft-hosted agents that provide automated deployment, teardown and re-deployment of typical Azure PaaS-based solutions.
+
+The challenge here is to balance the benefits of having a fully-automated process with the need for adequate security controls to minimise the risk of a pipeline becoming the source of a security incident.  Such benefits include:
+* 
```

Comment:
Did something go missing here?

Reply:
Yes 🤦  it helps to actually push ones changes!